Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/pages/LoginPage.java

```diff
@@ -0,0 +1,33 @@
+package com.automattic.simplenote.smoke.pages;
+
+import com.automattic.simplenote.R;
+import com.automattic.simplenote.smoke.utils.TestUtils;
+
+import static androidx.test.espresso.action.ViewActions.click;
+
+public class LoginPage extends BasePage {
+
+    private static final Integer INPUT_EMAIL = R.id.input_email;
+    private static final Integer INPUT_PASSWORD = R.id.input_password;
+    private static final Integer BUTTON_LOGIN = R.id.button;
+    private static final Integer ALERT_LOGIN = R.id.alertTitle;
+
+
+    public void login(String email, String password) {
+
+        enterTextInCustomInput(INPUT_EMAIL, email);
+        enterTextInCustomInput(INPUT_PASSWORD, password);
+
+        getViewById(BUTTON_LOGIN).perform(click());
+
+        TestUtils.idleForAShortPeriod();
+    }
+
+    public void checkLoginFailedMessage() {
+        waitForElementToBeDisplayed(R.id.alertTitle);
+    }
```

Comment:
It looks like this isn't used anywhere and is very similar to the `isLoginFailed()` method below; is it needed?

Reply:
`isLoginFailed` method has a try-catch block for handling a failed login attempt. But checkLoginFailedMessage especially waits for login to be failed. Hence separating them would be better.